pymilvus/orm/schema.py

```diff
@@ -10,66 +10,188 @@
 # or implied. See the License for the specific language governing permissions and limitations under
 # the License.
 
-import copy
 from typing import List
 import pandas
 from pandas.api.types import is_list_like
 
-from .constants import COMMON_TYPE_PARAMS
 from .types import DataType, map_numpy_dtype_to_datatype, infer_dtype_bydata
 from ..client.configs import DefaultConfigs
 from ..exceptions import (
+    MilvusException,
     CannotInferSchemaException,
     DataTypeNotSupportException,
     PrimaryKeyException,
     FieldsTypeException,
-    FieldTypeException,
     AutoIDException,
     ExceptionsMessage
 )
 
 
-class CollectionSchema:
-    def __init__(self, fields, description="", **kwargs):
-        if not isinstance(fields, list):
-            raise FieldsTypeException(0, ExceptionsMessage.FieldsType)
-        self._fields = [copy.deepcopy(field) for field in fields]
-        primary_field = kwargs.get("primary_field", None)
-        for field in self._fields:
-            if not isinstance(field, FieldSchema):
-                raise FieldTypeException(0, ExceptionsMessage.FieldType)
-            if primary_field == field.name:
-                field.is_primary = True
-        self._primary_field = None
-        for field in self._fields:
-            if field.is_primary:
-                if primary_field is not None and primary_field != field.name:
-                    raise PrimaryKeyException(0, ExceptionsMessage.PrimaryKeyOnlyOne)
-                self._primary_field = field
-                primary_field = field.name
-
-        if self._primary_field is None:
-            raise PrimaryKeyException(0, ExceptionsMessage.PrimaryKeyNotExist)
+class FieldSchema:
+    name: str
+    auto_id: bool = False
+    is_primary: bool = False
```

Comment:
Why these three static variables should be added? I thought maybe they shouldn't share between `FieldSchema`.

Reply:
This is initialization and a type hint for the non-property fields and it'll be handy for the type checks to be added later.

Also, I'm not fond of the error when we try to get `is_primary` from non-primary field.

```python
>>> class abc:
...     yx: str = ""
...
>>> abc = abc()
>>> abc.yx
''
>>> abc.ljq
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
AttributeError: 'abc' object has no attribute 'ljq'
>>>
```